src/set_theory/ordinal/fixed_point.lean

```diff
@@ -96,7 +92,7 @@ begin
   unfold nfp_family,
   rw @is_normal.sup _ H _ _ ⟨[]⟩,
   apply le_antisymm;
-  refine ordinal.sup_le (λ l, _),
+  refine sup_le (λ l, _),
```

Comment:
That doesn't sound like a good idea. Can you instead protect `ordinal.sup_le`, and all other lemmas that might conflict if you `open ordinal`?

Reply:
Ideally we'd entirely get rid of `ordinal.sup` and exclusively use `supr` and `Sup`. This would get rid of a lot of duplicate lemmas like this one. Not sure what that entails about this particular change.

Reply:
This specific change worsens the situation IMO, so I would drop it.